<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_contacts</class>
 <widget class="QMainWindow" name="main_contacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 208, 208);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 99, 149);
font: 75 10pt &quot;Quicksand&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="view_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 99, 149);
font: 75 10pt &quot;Quicksand&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>VIEW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upd_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 99, 149);
font: 75 10pt &quot;Quicksand&quot;;border-radius:10px;</string>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 99, 149);
font: 75 10pt &quot;Quicksand&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clr_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 99, 149);
font: 75 10pt &quot;Quicksand&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>CLEAR ALL</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
